tooling/api/src/main/java/org/jboss/windup/tooling/ToolingModeData.java

```diff
@@ -0,0 +1,18 @@
+package org.jboss.windup.tooling;
+
+import java.io.File;
+import java.util.List;
+import java.util.Set;
+
+public class ToolingModeData 
```

Comment:
Interesting debate can start here...... To use beans or not to use them..... I prefer to use them as a convention and use lombok in order to have the code less verbose.

Reply:
This is more of a convenience object instead of passing a forming a list of parameters that need to be parsed over and over. I don't see it necessary to get into a debate :-)